contexts/WalletManagerContext.tsx

```diff
@@ -38,6 +40,7 @@ type WalletManagerContextType = {
   wallet: WalletInfo | undefined;
   account: Account | undefined;
   addresses: string[];
+  tokens: TokenListItem[];
```

Comment:
I would move it to another context to make `WalletManager` only responsible for wallet management.

Reply:
I definitely want to talk about this. I felt the same way, but I had this thinking: the wallet manager already handles the balance of KAS, so why will it be different for tokens?

But in the end, I think you are right. If we decouple KAS and KRC20, in the code only, the UI should remain consistent, the code will be more maintainable, and we will be able to drop KRC20 if the implementation changes. 
But we will introduce code repetition (at least for the UI part).

_I am going to hit you up on slack and I will make the changes accordingly._

Reply:
Okay, your other comments convinced me already 🤣.

Reply:
23c79e872e35c519e6cc2b06e5921faf00be7d79

Moved to the dashboard inside this commit. I haven't made any context SWR is already having all the required features.